quickstarts/aws/aws-enhanced-monitoring/config.yml

```
@@ -0,0 +1,37 @@
+slug: aws-lambda-sam
+title: AWS Enhanced Monitoring
+description: |-
+  ## What is AWS Enhanced Monitoring?
+
+  Zero-administration compute platform for back-end web developers that runs
+  your code for you in AWS cloud.
+
+
+  ### Get started!
+
+  Start monitoring AWS Enhanced Monitoring by connecting Amazon Web Services (AWS) to New Relic!
+
+  Check out our AWS Enhanced Monitoring documentation to instrument your cloud service and manage the stability, scalability, and reliability of your systems with New Relic's infrastructure monitoring capabilities.
+
+  ### More info
+
+  Check out the [documentation](https://docs.newrelic.com/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-lambda-monitoring-integration/) to learn more about New Relic monitoring for AWS Enhanced Monitoring.
+
+summary: Monitor AWS Enhanced Monitoring by connecting AWS to New Relic
+level: New Relic
+authors: New Relic
```

I believe `authors` needs to be a list:
```suggestion
authors:
  - New Relic
```